tests/p2p/config.yaml

```diff
@@ -30,7 +30,7 @@ bootstrap:
     hare-wakeup-delta: '10'
   deployment_type: deployment # default:deployment  
 client:
-  image: 'spacemeshos/go-spacemesh-dev:go_auto'
+  image: 'spacemeshos/go-spacemesh-dev:develop'
```

Comment:
go-spacemesh:develop

Reply:
fixed in 207fd37bd289dcc691ce4b4207a102bc0f009af7